### PR TITLE
add wp bio

### PR DESCRIPTION
**Pull request by wpates**

### CODE REVIEW

File: roni/index.html [hunk 1]
```diff
@@ -3,13 +3,11 @@
 
 <head>
```
Comment (schrags08) [inline roni/index.html]: @wpates you are missing an opening HTML tag.

Comment (schrags08) [inline roni/index.html]: @wpates please disregard. operator error.

File: roni/index.html [hunk 2]
```diff
@@ -3,13 +3,11 @@
...
 </head>
 
 <body>
-    <h1>What's Next?</h1>
-    <p>Well, had to do a rush job. </p>
-    <h2>Learned A Ton!</h2>
-    <p>I used the command line to clone. Then used Code for the branching and commiting.</p>
-</body>
+    <h1>About my friend Whitney</h1>
+    <p>Whitney has 3 sisters. That's a lot of girl power.She's the only sister who has brown hair.</p>
```
Comment (schrags08) [inline roni/index.html]: @wpates, I think you can do better than this. It's pretty boring. Have you considered adding an image?

Comment (wpates) [inline roni/index.html]: I did. Thanks for the feedback, champ.

Comment (schrags08) [inline roni/index.html]: @wpates after consideration, what did you decide?

**Pull request merged**